guidelines/README.md

```diff
@@ -0,0 +1,5 @@
+# Guidelines
+
+The objective of guidelines is to share knowledge between club leaders in a more
```

Comment:
I don't understand what this paragraph is trying to tell me. @MaxWofford could you please try to reword? Happy to talk about it as well.